### DISCUSSION TITLE
Multiple windows

### DISCUSSION THREAD

**Discussion by amnykon** [raysan5/raylib#3389] [Features]
I am planning on making several 4 player games targeting the Raspberry pi (ideally I would  port them to other OSes). Raspberry pi has 2 mini HDMI outputs, so I would like to create a separate window for the 2nd screen. I agree the 2nd screen should be optional.
I have seen the discussion in https://github.com/raysan5/raylib/issues/1989. A select window context is a good way to implement this feature without adding complication to the library.  Could we implement this functionality to the main library, so I would not need to maintain the code in a separate fork?

Would you consider merging a PR adding this functionality if I made one?

**Comment by raysan5** [comment]
@amnykon Sorry, no plans to add it to the main library for now. Feel free to fork it and add it for your specific needs.
